lib/DDG/Spice/Dockerhub.pm

```diff
@@ -0,0 +1,30 @@
+package DDG::Spice::Dockerhub;
+# ABSTRACT: Search for Docker Hub images
+
+use DDG::Spice;
+use strict;
+use warnings;
+
+spice is_cached => 1;
+spice proxy_cache_valid => '200 1d';
```

Comment:
Maybe just my opinion but it might not be a very good idea to cache this, images and information could change rather quickly.